src/video-player.js

```diff
@@ -368,6 +372,12 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     }
   }
 
+  _initPictureInPicture() {
+    if (!this.playerOptions.pictureInPictureToggle && this.videojs.controlBar) {
+      this.videojs.controlBar.removeChild('pictureInPictureToggle');
```

Comment:
just making sure i understand, we have to do this because the default for videojs is true right?